pkg/apis/eventing/v1beta1/trigger_conversion_test.go

```diff
@@ -22,7 +22,7 @@ import (
 )
 
 func TestTriggerConversionBadType(t *testing.T) {
-	good, bad := &Trigger{}, &Trigger{}
+	good, bad := &Trigger{}, &Broker{}
```

Comment:
There's not much value in this conversion test and other conversion tests for other types unless we add a test for good conversion case as well.

You didn't have to change the `&Trigger` to `&Broker` above to make the conversion fail and tests pass. That's what I mean why there's not much value.

But, let's ignore this for now and I am interested in having proper conversion tests in a later PR.

Reply:
Yes, agreed :)

Reply:
Created https://github.com/knative/eventing/pull/3395